src/scripts/release_picard.sh

```diff
@@ -184,12 +184,12 @@ cd $TMPDIR
 # clone
 git clone $PICARDGITROOT picard
 cd picard
-ant clone-htsjdk
+ant -buildfile build-from-src.xml clone-htsjdk
```

Comment:
Is it technically possible to decouple htsjdk and picard github releases as things stand now, or would that be hard?